### PR TITLE
Added asset and extras to runtime abstraction

### PR DESCRIPTION
**Pull request by kcoley**
Added Asset to runtime abstraction

### CODE REVIEW

File: Source/Program.cs [hunk 1]
```diff
@@ -140,24 +136,27 @@ private static void Main(string[] args)
...
+                    var gltf = new Gltf
+                    {
+                        Asset = asset.ConvertToAsset()
+                    };
+
                     var dataList = new List<Data>();
 
                     var geometryData = new Data(test.ToString() + "_" + comboIndex + ".bin");
                     dataList.Add(geometryData);
                     Runtime.GLTF wrapper = Common.SinglePlane(gltf, geometryData);
```
Comment (stevk) [inline Source/Program.cs]: I'm not sure why this line compiled, but I'm pretty sure `gltf` needs to be replaced by `asset` here.

There also is a similar line near the end of the file `glTFLoader.Interface.SaveModel(gltf, assetFile);`

Comment (kcoley) [reply Source/Program.cs]: Good catch, I'll fix that

Comment (kcoley) [reply Source/Program.cs]: Turns out we are not even using gltf and geometryData in Common anymore.  I'll make a separate PR to remove those arguments from SimplePlane and other models.

Comment (kcoley) [reply Source/Program.cs]: I rebased the update to remove `gltf` and `geometryData` from the Common args.

**Pull request merged**